### PR TITLE
STY: fix whitespace around keywords

### PR DESCRIPTION
**Pull request by tacaswell**
Fixes new style checks.

Due to changes in linting tools these are now being flagged.

### CODE REVIEW

**Comment by danielballan** [comment]
Unit tests are failing.

**Comment by tacaswell** [comment]
The failures are ophyd dropping support for 3.6 and 3.7 still.

File: bluesky/tests/test_examples.py [hunk 1]
```diff
@@ -451,13 +451,13 @@ def test_duplicate_keys(RE, hw):
...
 
     def gen():
-        yield(Msg('open_run'))
-        yield(Msg('create', name='primary'))
-        yield(Msg('trigger', hw.det))
-        yield(Msg('trigger', hw.identical_det))
-        yield(Msg('read', hw.det))
-        yield(Msg('read', hw.identical_det))
-        yield(Msg('save'))
+        yield (Msg('open_run'))
```
Comment (mrakitin) [inline bluesky/tests/test_examples.py:454]: Does it need the parentheses around `Msg` in this case?

Comment (tacaswell) [reply bluesky/tests/test_examples.py:454]: No, but they do no harm.

Comment (prjemian) [reply bluesky/tests/test_examples.py:454]: No _technical_ harm.  They do cause minor obfuscation, such as this question.  Better to remove if not strictly needed.

**Pull request merged**